Tools/Scripts/libraries/webkitscmpy/webkitscmpy/test/setup_unittest.py

```diff
@@ -222,3 +222,28 @@ def test_commit_message(self):
 ''')
             finally:
                 sys.path.remove(os.path.dirname(pcm))
+
+    def test_security_levels(self):
+        with OutputCapture(level=logging.INFO), mocks.local.Git(self.path, remote='git@example.org:project/project') as git, mocks.local.Svn():
+            with open(os.path.join(self.path, '.git', 'config'), 'a') as f:
+                f.write('[webkitscmpy "remotes"]\n')
+                f.write('    origin = git@example.org:project/project\n')
+                f.write('    security = git@example.org:project/project-security\n')
```

Comment:
Per my previous comment, what happens if you reverse `origin` and `security`?  Will `security` be at a lower security level than `origin`?